src/lib/functions/netlify-function.ts

```diff
@@ -26,6 +25,29 @@ const getNextRun = function (schedule) {
 export default class NetlifyFunction {
   public readonly name: string
   public readonly mainFile: string
+  public readonly displayName: string
+  public readonly schedule?: string
+
+  private readonly directory: string
+  private readonly projectRoot: string
+  private readonly blobsContext: BlobsContext
+  private readonly timeoutBackground: number
+  private readonly timeoutSynchronous: number
+
+  // Determines whether this is a background function based on the function
+  // name.
+  private readonly isBackground: boolean
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private buildQueue?: Promise<any>
```

Comment:
Yes, we still plan on typing this, but not in this PR. I tried `@ts-expect-error`, but it will fail because the line isn't a TSC error. Using `any` is forbidden by the linter, so we'll have to disable the lint rule, not the type checker.

Reply:
So how will we know that this is something we still need to type, as opposed to a legitimate use of `any` that we want to keep?

Reply:
ideally, i'd like to add a "FIXME" comment to this eslint disable thing - but eslint will error about that. maybe I can add it one line above, let me try

Reply:
how do you like https://github.com/netlify/cli/pull/6211/commits/d4e5e364e96891ce204a08d2c8f23890435cd71c?

Reply:
i've seen that pattern in other codebases, and I like it!

Reply:
LGTM!